Makefile

```diff
@@ -165,19 +165,35 @@ shellcheck: $(SH_FILES) ## Run checks for shell scripts.
 
 ##@ Docker
 
+TMP_DIR!=mktemp -d
```

Comment:
In the Makefile the `!=` is a shell assignment operator.
You can read about it more here: https://www.gnu.org/software/make/manual/html_node/Setting.html

It basically runs that as a shell command and sets the output as the variable value.

The difference between using `!=` and `=$(shell <command>)` is that the second one would be run everytime the variable is accessed, while the first one only runs once when the command is called.

In case of the `maketmp -d` the second one would result in the temp folder being different everytime that `$(TMP_FILE)` was accessed, while the first one results in it always being the same folder.